src/internal/Esp32_i2s.c

```diff
@@ -156,8 +156,12 @@ bool i2sInitDmaItems(uint8_t bus_num) {
         item->free_ptr = NULL;
         item->buf = NULL;
     }
-    itemPrev->eof = 1;
-    item->eof = 1;
+    if (itemPrev) {
```

Comment:
Is there some other way to suppress the warning than by adding code here?

There is no possible way to get to this point with these being NULL.

Reply:
It's not needed to remove the warning, just wanted to be thorough.